core/src/main/scala/org/apache/spark/h2o/backends/SharedBackendUtils.scala

```diff
@@ -121,6 +121,7 @@ private[backends] trait SharedBackendUtils extends Logging with Serializable {
       .add("-stacktrace_collector_interval", Some(conf.stacktraceCollectorInterval).filter(_ > 0))
       .add("-nthreads", Some(conf.nthreads).filter(_ > 0).orElse(conf.sparkConf.getOption("spark.executor.cores")))
       .add("-internal_security_conf", conf.sslConf)
+      .add("-client_disconnect_timeout", conf.clientCheckRetryTimeout)
```

Comment:
Nit: the naming does not match: disconnect v. retry.

Reply:
Yup, agree the naming is not ideal :( but it is already used it the same in external backend